app/controllers/events_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
   def create
     @event = Event.new(event_params)
     if @event.save
-      redirect_to root_path, notice: "Event was successfully created." # please could we redicrect to @event page if an event is successfully created?
+      redirect_to event_path @event, notice: "Event was successfully created." # please could we redicrect to @event page if an event is successfully created?
```

Comment:
Does it work like this? Dote it not need to have @event in brackets? redirect_to event_path(@event)

Reply:
Yes, it does with ruby you don't need the parenthesis there if it is one param.